draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1167,18 +1176,16 @@ uCDN-A                         dCDN-B                         dCDN-C
    <vspace blankLines="1"/>
    Value: Array of GenericTriggerExtension objects (see <xref target="generic-extension-object"/>).
    <vspace blankLines="1"/>
-   Mandatory: No. The default is no extensions.
+   Mandatory: No. The default is no extensions. 
    </t>
 
    <t hangText="Name: labels">
    <vspace blankLines="1"/>
    Description: Array of trigger labels.
    <vspace blankLines="1"/>
-   Value: Array of Trigger labels, Each label is encoded as a JSON string and SHOULD be unique per trigger.
-
-
+   Value: Array of Trigger labels. Each label is encoded as a JSON string and SHOULD be unique per trigger.
```

Comment:
Re: Is there a better wording for this you can think of?

Maybe, "Each label is encoded as a JSON string and should appear only once in the array of trigger labels."  Or, "Each label is encoded as a JSON string and should be unique within the array of trigger labels."  The current text seems to say that the label should be unique per trigger, when it is really unique inside the list of trigger labels for each trigger.

Reply:
Went with the 2nd option